client/my-sites/site-settings/settings-p2/preapproved-domains.jsx

```diff
@@ -1,285 +0,0 @@
-import { Card, FormLabel } from '@automattic/components';
```

Comment:
Is this p2-specific functionality still available somewhere else? I'm wondering if not, whether access to this should have remained but it was removed accidentally.